### PR TITLE
koordlet: report the amount of reclaimable resources of the node

### PR DESCRIPTION
**Pull request by hormes**
Part of #1361

### CODE REVIEW

File: pkg/koordlet/prediction/peak_predictor.go [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+	podRequests := util.GetPodRequest(pod, v1.ResourceCPU, v1.ResourceMemory)
+	podCPURequest := podRequests[v1.ResourceCPU]
+	podMemoryRequest := podRequests[v1.ResourceMemory]
+
+	reclaimableCPUMilli := int64(0)
+	reclaimableMemoryBytes := int64(0)
+
+	if p95CPU, ok := p95Resources[v1.ResourceCPU]; ok {
+		peakCPU := util.MultiplyMilliQuant(p95CPU, 1.1)
```
Comment (saintube) [inline pkg/koordlet/prediction/peak_predictor.go:141]: This ratio can be inaccurate and not universal. How about adding a parameter to adjust it when necessary?

Comment (hormes) [reply pkg/koordlet/prediction/peak_predictor.go:141]: later add a pr separately to add configurations.

**Review by saintube — APPROVED** [review]
/lgtm

**Pull request merged**